org.damask_multiphysics.DAMASK_grid.yaml

```diff
@@ -0,0 +1,68 @@
+id: org.damask_multiphysics.DAMASK_grid
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: DAMASK_grid
+finish-args:
+  - --filesystem=xdg-documents
+
+modules:
+  - name: grid_petsc
+    buildsystem: simple
+    build-options:
+      env:
+        PETSC_ARCH: "gfortran-openmpi"
+        PETSC_DIR: "/run/build/grid_petsc"
+    build-commands:
+      - ./configure --with-shared-libraries=1 --with-mpi-f90module-visibility=0
+        --prefix=/app --with-packages-download-dir=/run/build/grid_petsc
+        --download-openmpi=1 --download-fblaslapack=1 --download-hdf5=1 --with-hdf5-fortran-bindings=1 --download-fftw=1
+        && make all install
+    sources:
+      - type: archive
+        url: https://web.cels.anl.gov/projects/petsc/download/release-snapshots/petsc-3.21.4.tar.gz
+        sha256: a9ae076d4617c7d84ce2bed37194022319c19f19b3930edf148b2bc8ecf2248d
+      - type: file
+        url: https://download.open-mpi.org/release/open-mpi/v5.0/openmpi-5.0.3.tar.gz
+        sha256: 3e12160dd1d6f5cd8897bbab63a05fb9223a1e20a1726db23607aac25dc1aa19
+      - type: file
+        url: https://bitbucket.org/petsc/pkg-fblaslapack/get/v3.4.2-p3.tar.gz
+        sha256: 644629f3559bf7ecdb8d01975131f982b15da823a39c5df4fa5764a4a5e9b34a
+      - type: file
+        url: https://web.cels.anl.gov/projects/petsc/download/externalpackages/hdf5-1.14.3-p1.tar.bz2
+        sha256: ec0da2dd048b4933603e810e1a341ba497a1e0b464dc3118549f38307945ff6c
+      - type: file
+        url: https://www.fftw.org/fftw-3.3.10.tar.gz
+        sha256: 56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467
+
+  - name: grid_libfyaml
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh && ./configure --prefix=/app && make && make install
```

Comment:
this should be `buildsystem: autotools`

Reply:
Remove the autotools commands, prefix is set automatically when you use `buildsystem: autotools`.

Reply:
fixed (seems to work well without `.bootstrap.sh`